### PR TITLE
Coverage

### PR DESCRIPTION
**Pull request by rleonid**
Provide coverage capabilities to testing driver!

### CODE REVIEW

**Comment by smondet** [comment]
Can you elaborate/document what the `joiner.ml` is, and why it is copied from Kaputt?

**Comment by rleonid** [comment]
@smondet Updated the documentation.

File: tools/travis_ci_test.sh [hunk 1]
```diff
@@ -64,3 +64,8 @@ make
 
 echo Testing
 make test
+
+echo PostingCoverage
+opam install ocveralls
+cd _driver
+ocveralls --repo_token $COVERALLSTOKEN --send ../bisect0001.out
```
Comment (smondet) [inline tools/travis_ci_test.sh:71]: Where does that variable come from?

**Comment by smondet** [comment]
Apart from that question, LGTM !

**Comment by rleonid** [comment]
The environment variable is set by Travis to automatically post to [Coveralls](https://coveralls.io/r/rleonid/oml)

**Pull request merged**